src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -1129,9 +1115,8 @@ private void AddParameterRefKindIfNeeded(IParameterSymbol symbol)
         {
             if (format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
             {
-                // https://github.com/dotnet/roslyn/issues/61647: Use public API.
-                if ((symbol as Symbols.PublicModel.ParameterSymbol)?.GetSymbol<ParameterSymbol>() is { } parameter &&
-                    parameter.EffectiveScope == DeclarationScope.RefScoped &&
+                if (symbol.ScopedKind == ScopedKind.ScopedRef &&
+                    (symbol as Symbols.PublicModel.ParameterSymbol)?.GetSymbol<ParameterSymbol>() is { } parameter &&
                     !ParameterHelpers.IsRefScopedByDefault(parameter) &&
```

Comment:
Are we missing ```IsRefScopedByDefault``` as a public API? Is there an alternative way to detect this in a language independent way?

Reply:
If so, perhaps we should use this condition only for C# symbols and not dismiss other symbols?

Reply:
`IsRefScopedByDefault()` relies on `ModuleSymbol.UsesUpdatedEscapeRules`, which is not tracked in VB, nor is `scoped`. Changed to use for C# symbols only.